helpers.go

```diff
@@ -1,24 +1,28 @@
 package main
 
 import (
-	"math/rand"
 	"time"
 
+	// xrand "golang.org/x/exp/rand"
```

Comment:
```suggestion
```
wasn't sure if this was one of those odd things like cgo that was helping with something or just from tinkering

Reply:
Good catch, I was reading up on some of the exp/rand changes to seed sourcing and tested it out, I did not any significant difference so figured it'd be better to stick with the standard lib here.